### PR TITLE
Remove all dependencies on legacy telemetry

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

**Review by greg-dennis — APPROVED** [review]

File: internal/fptest/log.go [hunk 1]
```diff
@@ -156,38 +116,6 @@ func LogQuery(t testing.TB, what string, query LoggableQuery, obj ygot.Validated
 	logQuery(t, what, query, obj, true)
 }
 
-// WriteYgot is like LogYgot but only writes to test outputs dir so it
-// does not pollute the test log.
-func WriteYgot(t testing.TB, what string, path ygot.PathStruct, obj ygot.ValidatedGoStruct) {
```
Comment (liulk) [inline internal/fptest/log.go:161]: Although not currently used, add a WriteQuery?

**Review by liulk — COMMENTED** [review]
Mostly LGTM. Thanks for adding the LogQuery beforehand.

**Review by liulk — APPROVED** [review]

**Comment by liulk** [comment]
Can you resolve the conflict?

**Comment by DanG100** [comment]
rebased

**Review by liulk — APPROVED** [review]

**Pull request merged**